client/post-editor/editor-author/index.jsx

```diff
@@ -41,10 +41,15 @@ export class EditorAuthor extends Component {
 		const name = author.display_name || author.name;
 		const Wrapper = this.userCanAssignAuthor() ? AuthorSelector : 'div';
 		const popoverPosition = touchDetect.hasTouch() ? 'bottom right' : 'bottom left';
+		const wrapperProps = this.userCanAssignAuthor() && {
+			siteId: post.site_ID,
+			onSelect: this.onSelect,
+			popoverPosition,
+		};
```

Comment:
Should we be using the spread operator on a boolean value? `false && {}` is `false